src/applicant/challenge.md

```diff
@@ -1,30 +1,38 @@
 # Code Challenge
 
-Implement the following call to action component. Sadly our designers forgot about the mobile design. Get creative and implement a responsive solution that works on all viewport sizes. 
+Implement the following call to action component. Sadly our designers forgot about the mobile design. Get creative and implement a responsive solution that works on all viewport sizes.
 
 ![cta-open.png](../assets/cta-open.png)
 
-### Specifications
-- Font Sizes:
-  - Headline: 36px
-  - Call to action title: 24px
-  - Call to action description: 16px
-  - Button text : 12px
-  - Contact box title: 24px
+### Design Requirements
 
-- Fonts (Google Fonts)
-  - Serif: Playfair Display
-  - Sans Serif: Roboto Condensed
+-   Font Sizes:
 
-- There are no measurements for this Screen
-  - Try to measure by eye. (It will not be evaluated if it is a pixel perfect match)
+    -   Headline: 36px
+    -   Call to action title: 24px
+    -   Call to action description: 16px
+    -   Button text : 12px
+    -   Contact box title: 24px
 
-- Use any asset or mock image as you like
-- The content next to the image is always vertical centered in the grey background
-- The whole component has full page width
-- A click on the call to action button opens the contact box
-- Add a close button for the contact box wherever you think it would make sense from a UX standpoint
-- A click on the close button closes the contact box
+-   Fonts (Google Fonts)
 
-Good luck and have fun!
+    -   Serif: Playfair Display
+    -   Sans Serif: Roboto Condensed
+
+-   There are no measurements for this Screen
+
+    -   Try to measure by eye. (It will not be evaluated if it is a pixel perfect match)
+
+-   Use any asset or mock image as you like
+-   The content next to the image is always vertical centered in the grey background
+-   The whole component has full page width
+-   A click on the call to action button opens the contact box
+-   Add a close button for the contact box wherever you think it would make sense from a UX standpoint
+-   A click on the close button closes the contact box
 
+### Code Requirements
+
+-   For CSS try to use as less classes as possible.
```

Comment:
what is the goal of this requirement, to write as many semantic css selectors as possible?

Reply:
For me there are different perspectives:
1. I would like to give some requirements from a code perspective (not only design), to see how they were handled.
2. It reduces the chance, that someone copies code from CSS frameworks.

On the other hand it shows some character, if people just make it super clean or uses BEM or Utility Classes and it's nice to discuss that stuff.

What do you think – should I kick this line again? @SheepFromHeaven @ctmm

Reply:
I agree with Mario's impulse. 
I would even go as far as recommending to candidates that they do not use any CSS frameworks – (tailwind, bootstrap, bulma, etc).

I have nothing against using CSS frameworks on production apps.
However, **this is a coding exercise to hire candidates.**

If we're hiring frontend engineers, I expect that the challenge provides insight into the candidate's expertise on CSS.
This way we can more accurately access if the candidate has the expected knowledge/experience for the position they're applying to.